conf/log.conf

```diff
@@ -1,19 +1,30 @@
 log4perl.rootLogger= sub {return ($ENV{LOG_LEVEL_ROOT} // "ERROR") . ", LOGFILE";}
 log4perl.logger.mongodb= sub {return ($ENV{LOG_LEVEL_MONGODB} // "INFO") . ", MONGODB_LOGFILE";}
 log4perl.logger.ratelimiter = sub {return ($ENV{LOG_LEVEL_RATE_LIMITER} // "INFO") . ", RATELIMITER_LOGFILE";}
+log4perl.logger.requeststats = sub {return ($ENV{LOG_LEVEL_REQUEST_STATS} // "INFO") . ", REQUESTSTATS_LOGFILE";}
 log4perl.PatternLayout.cspec.S = sub { my $context = Log::Log4perl::MDC->get_context; use Data::Dumper (); local $Data::Dumper::Indent    = 0; local $Data::Dumper::Terse     = 1; local $Data::Dumper::Sortkeys  = 1; local $Data::Dumper::Quotekeys = 0; local $Data::Dumper::Deparse= 1; my $str = Data::Dumper::Dumper($context); $str =~ s/[\n\r]/ /g; return $str; }
+log4perl.PatternLayout.cspec.J = sub {  my $context = Log::Log4perl::MDC->get_context; use JSON::PP; my $json_utf8 = JSON::PP->new->utf8(1)->allow_nonref->canonical; my $str = $json_utf8->encode($context); return $str; }
```

Comment:
Great !

Why not use JSON::MaybeXS instead of JSON::PP ? It's really faster (I made so that JSON::XS is installed)

Reply:
Good point

Reply:
Or use `JSON` as it's a compatible wrapper for both